enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -409,6 +409,23 @@ class SubsidyAccessPolicyCanRedeemRequestSerializer(serializers.Serializer):
     )
 
 
+# pylint: disable=abstract-method
+class SubsidyAccessPolicyCanRequestRequestSerializer(serializers.Serializer):
+    """
+    Request serializer to validate can_request endpoint query params.
+
+    For view: SubsidyAccessPolicyRequestViewset.can_request
+    """
+    content_key = ContentKeyField(
+        required=True,
+        help_text='Content key about which requestability will be queried.',
+    )
+    lms_user_id = serializers.IntegerField(
+        required=False,
+        help_text='The user identifier for which requestability will be queried (only applicable to staff users).',
```

Comment:
Question: why is it only applicable to staff users?

Reply:
So that it behaves just like `can_redeem`

Reply:
https://github.com/openedx/enterprise-access/blob/55eaa5d2f2dbf3fe4004a81d8e12ee518209cda1/enterprise_access/apps/api/v1/views/subsidy_access_policy.py#L766